layers/gre_test.go

```diff
@@ -253,3 +254,138 @@ func BenchmarkEncodePacketEthernetOverGRE(b *testing.B) {
 		buf.Clear()
 	}
 }
+
+var testGREChecksum = map[uint16][]gopacket.SerializableLayer{
+	0x77ff: {
```

Comment:
has at least one of these come from an in-the-wild GRE packet?  If not, maybe we can see if we can rustle one up somewhere.

Just want to make absolutely sure that the checksums we're generating actually match in-the-wild checksums.